src/components/Chip/Chip.tsx

```diff
@@ -89,27 +89,32 @@ const Chip = ({
     </>
   );
 
-  const chipClassName = classNames({
-    [`p-chip--${appearance}`]: !!appearance,
-    "p-chip": !appearance,
-  });
+  const chipClassName = classNames(
+    {
+      [`p-chip--${appearance}`]: !!appearance,
+      "p-chip": !appearance,
+    },
+    props.className
+  );
 
   if (onDismiss) {
     return (
-      <span className={chipClassName} aria-pressed={selected} {...props}>
+      <span {...props} className={chipClassName}>
```

Comment:
I'm not sure of the implications of being able to spread `HTMLProps<HTMLButtonElement>` onto a `<span>`... but if TS is fine with it I guess it's ok? The alternative would be to set it up so that if `onDismiss` exists then button props are allowed, otherwise span props are allowed etc but that might be more effort than its worth